src/SignalR/clients/cpp/src/signalrclient/hub_connection_impl.cpp

```diff
@@ -270,27 +244,11 @@ namespace signalr
                 {
                     set_exception(
                         std::make_exception_ptr(
-                            signalr_exception(message.at(_XPLATSTR("error")).serialize())));
+                            hub_exception(message.at(_XPLATSTR("error")).serialize())));
                 }
 
                 set_result(json::value::null());
             };
         }
-
-        static utility::string_t adapt_url(const utility::string_t& url, bool use_default_url)
```

Comment:
Why was this needed before but not now?

Reply:
I think I removed it's usage a long time ago when I ported the client. It's not referenced anywhere which is why I removed it here.